modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -2160,6 +2162,73 @@ test.describe('Manage object entries through View Object Entries', () => {
 			viewObjectEntriesPage.page.getByText('astronaut.png')
 		).toBeVisible();
 	});
+
+	test('verify that an appropriate error message appears after attempting to upload an oversized file', async ({
```

Comment:
can you sort this test? I believe it should be before this one: `test('Verify that temporary files are deleted from the database if the object creation is not completed',`